dev/test-dependencies.sh

```diff
@@ -85,10 +85,15 @@ for HADOOP_PROFILE in "${HADOOP_PROFILES[@]}"; do
 
   echo "Generating dependency manifest for $HADOOP_PROFILE"
   mkdir -p dev/pr-deps
-  $MVN $HADOOP2_MODULE_PROFILES -P$HADOOP_PROFILE dependency:build-classpath -pl assembly \
-    | grep "Dependencies classpath:" -A 1 \
-    | tail -n 1 | tr ":" "\n" | rev | cut -d "/" -f 1 | rev | sort \
-    | grep -v spark > dev/pr-deps/spark-deps-$HADOOP_PROFILE
+  $MVN $HADOOP2_MODULE_PROFILES -P$HADOOP_PROFILE dependency:build-classpath -pl assembly | grep "Dependencies classpath:" -A 1 | tail -n 1 | tr ":" "\n" | awk -F '/' '{
```

Comment:
Nit: it would be great if you add comments about what is going on for the text processing here.

Reply:
Added comments above the text processing part.

Reply:
Let's keep the multi-line format. Especially, we don't need to change the old line 88 and 89.

Reply:
Adjusted the format.